src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { PostListProps } from '../../types';
+import { PostInfo } from '../PostInfo';
+
```

Comment:
you are passing a prop `posts` from `app.tsx ` , you have to use `interface` to mention type here before using it